plugins/outputs/influxdb_v2/http.go

```diff
@@ -166,7 +183,33 @@ func (c *httpClient) Write(ctx context.Context, metrics []telegraf.Metric) error
 		return nil
 	}
 
-	desc := resp.Header.Get("X-Influx-Error")
+	writeResp := &genericRespError{}
+	json.NewDecoder(resp.Body).Decode(writeResp)
+	var desc string
+
+	switch resp.StatusCode {
+	case http.StatusBadRequest: // 400
+		// LineProtocolError
+		desc = fmt.Sprintf("%s - %s;%d;%s", writeResp, writeResp.Op, writeResp.Line, writeResp.Err)
```

Comment:
At this point you probably only care about code, message and line; not op and err

Op/Err is useful as a stack trace

Reply:
Should op/err be removed from the spec for `Error` and `LineProtocolError` if it's not intended to be used anywhere?

Reply:
No it should not be removed the spec. The audience for op/err are people when debugging and doing more thorough analysis of a problem.